src/containers/CategoryFilter.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import categories from '../constants/categories';
+
+const CategoryFilter = (props) => {
```

Comment:
Our `CategoryFilter.js` should be placed in src/components directory.
For any file not connecting to the redux store should be in component folder. Components in Component folder do not have direct access to Redux or other data stores.
[read more](https://www.reactnative.guide/9-redux/9.2-presentational-vs-containers.html)